languages/go/pkg/host.go

```diff
@@ -0,0 +1,325 @@
+// """Translate between Polar and the host language (Python)."""
```

Comment:
I would probably split this out as a separate host `pkg`. e.g. `github.com/osohq/oso/languages/go/pkg/host`. (Potentially `github.com/osohq/oso/languages/go/internal/pkg/host`)